packages/front-end/components/Features/ExperimentRefSummary.tsx

```diff
@@ -0,0 +1,328 @@
+                />
+              </td>
+              <td>{r?.name}</td>
+              <td>
+                <div className="d-flex">
+                  <div
+                    style={{
+                      width: "4em",
+                      maxWidth: "4em",
+                      margin: "0 0 0 auto",
+                    }}
+                  >
+                    {percentFormatter.format(phase.variationWeights[j] || 0)}
+                  </div>
+                </div>
+              </td>
+            </tr>
+          ))}
+          <tr>
+            <td colSpan={4}>
+              <ExperimentSplitVisual
+                values={experiment.variations.map((v, i) => ({
+                  value: rule.variations[i]?.value || "",
+                  weight: phase.variationWeights[i] || 0,
+                  name: v.name,
+                }))}
+                coverage={effectiveCoverage}
+                label="Traffic split"
+                unallocated="Not included (skips this rule)"
+                type={type}
+                showValues={false}
+                stackLeft={true}
+                showPercentages={true}
+              />
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <div className="row align-items-center">
+        <div className="col-auto">
+          <strong>TRACK</strong>
+        </div>
+        <div className="col">
+          {" "}
+          the result using the key{" "}
+          <span className="mr-1 border px-2 py-1 bg-light rounded">
+            {experiment.trackingKey}
+          </span>{" "}
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function ExperimentRefEndBehavior({
+  experiment,
+  rule,
+  type,
+}: {
+  experiment: ExperimentInterfaceStringDates;
+  rule: ExperimentReferenceRule;
+  type: FeatureValueType;
+}) {
+  const results = experiment.results || "dnf";
+
+  // Won or lost, roll out winning variation to 100%
+  if (results === "won" || results === "lost") {
+    const variation = results === "won" ? experiment.winner || 1 : 0;
+    return (
+      <div className="row align-items-top">
+        <div className="col-auto">
+          <strong>SERVE</strong>
+        </div>
+        <div className="col">
+          <ValueDisplay
+            value={rule.variations[variation]?.value || ""}
+            type={type}
+          />
+        </div>
+      </div>
+    );
+  }
+
+  // TODO: dnf and inconclusive
+
+  return null;
+}
+
+function ExperimentRefPhaseHeader({
+  isOpen,
+  setOpen,
+  i,
+  start,
+  end,
+  label,
+  badge,
+  additionalInfo,
+  active,
+}: {
+  isOpen: boolean;
+  setOpen: (open: number) => void;
+  label: string;
+  i: number;
+  start?: string;
+  end?: string;
+  badge?: ReactElement;
+  additionalInfo?: ReactElement;
+  active?: boolean;
+}) {
+  return (
+    <a
+      href="#"
+      className="row mb-2"
+      onClick={(e) => {
+        e.preventDefault();
+        setOpen(isOpen ? -1 : i);
+      }}
+    >
```

Comment:
This is somewhat of an accessibility/semantics nit: It looks like based on this behaviour, it's better for this to be a button. The class `btn` allows us to use an unstyled button. It adds padding by default it looks like but we can do `p-0` if we don't want it.